src/auto/flags.rs

```diff
@@ -202,7 +202,9 @@ bitflags! {
     #[doc(alias = "OstreeRepoCommitTraverseFlags")]
     pub struct RepoCommitTraverseFlags: u32 {
         #[doc(alias = "OSTREE_REPO_COMMIT_TRAVERSE_FLAG_NONE")]
-        const REPO_COMMIT_TRAVERSE_FLAG_NONE = ffi::OSTREE_REPO_COMMIT_TRAVERSE_FLAG_NONE as u32;
```

Comment:
This is a semver-breaking change (though I'm not sure why codegen changed this time).

Reply:
Apparently this change is correct because the gir file gained an explicit member-name attribute now (which didn't exist in 2022.1), here:
https://github.com/ostreedev/ostree-rs/blob/70ba3219811c3ebd1bc14876c5ca687a4f48b3e1/gir-files/OSTree-1.0.gir#L12540-L12542

Reply:
OK.  So I would agree that *technically* this is a semver-breaking change.  But, I'd bet a not-small amount of money no one is actually using this flag.